product/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from product.views import CategoriesView, ProductsView, ProductDetailView
+
+urlpatterns = [
+    path('', CategoriesView.as_view()),
+    path('/categories', ProductsView.as_view()),
```

Comment:
상품 정보를 받아오는 엔드포인트이므로 자원(resource) 중심으로 엔드포인트를 수정해주세요!
```suggestion
    path('/products', ProductsView.as_view()),
```

Reply:
수정했습니다~